test/runtests.jl

```diff
@@ -22,3 +22,38 @@ end
 haskey(ENV, "CI") && GPUCompiler.timings()
 
 end
+
+using Pkg
+Pkg.add(PackageSpec(name="LLVM_full_jll", version="$(Base.libllvm_version.major)"))
```

Comment:
:sweat_smile:

Reply:
Yeah I need to check whether LLVM_jll has lit and the rest of the tools.